ios/Plugin/Plugin.swift

```diff
@@ -175,6 +175,17 @@ public class NativeAudio: CAPPlugin {
         audioAsset.setVolume(volume: volume as NSNumber)
         call.success()
     }
+
+    @objc func isPlaying(_ call: CAPPluginCall) {
```

Comment:
Also add definition in Plugin.m file for new method

Reply:
I did as you asked. Thanks for your comment.